docs/sources/about.md

```diff
@@ -76,33 +81,51 @@ You should run the Static mode Kubernetes operator when:
 
 ### Flow mode
 
-[Flow mode][] is a stable variant of Grafana Agent first introduced on
-September 29, 2022.
+[Flow mode][] is a stable variant of Grafana Agent, introduced on September 29, 2022.
 
-Flow mode was introduced as a re-imagining of Grafana Agent with a focus on
-vendor neutrality, ease-of-use, improved debuggability, and ability to adapt to
-the needs of power users by adopting a configuration-as-code model.
-
-Flow mode is considered to be the future of the Grafana Agent project.
+Grafana Agent Flow mode focuses on vendor neutrality, ease-of-use,
+improved debugging, and ability to adapt to the needs of power users by adopting a configuration-as-code model.
 
 You should run Flow mode when:
 
 * You need functionality unique to Flow mode:
 
-  * **Debuggability**: You need to more easily debug configuration issues using
-    a UI.
+  * **Improved debugging**: You need to more easily debug configuration issues using a UI.
+
+  * **Full OpenTelemetry support**: Support for collecting OpenTelemetry metrics, logs, and traces.
+
+  * **PrometheusRule support**: Support for the PrometheusRule resource from the Prometheus Operator project for configuring Grafana Mimir.
+
+  * **Ecosystem transformation**: You need to be able to convert Prometheus and Loki pipelines to and from OpenTelmetry Collector pipelines.
+
+  * **Grafana Pyroscope support**: Support for collecting profiles for Grafana Pyroscope.
+
+#### Core telemetry
+
+|              | Prometheus Agent mode | Grafana Agent Static mode | Grafana Agent Operator | OpenTelemetry Collector | Grafana Agent Flow mode  |
+|--------------|-----------------------|---------------------------|------------------------|-------------------------|--------------------------|
+| **Metrics**  | Prometheus            | Prometheus                | Prometheus             | OTel                    | [Prometheus][], [OTel][] |
+| **Logs**     | No                    | Loki                      | Loki                   | OTel                    | [Loki][], [OTel][]       |
+| **Traces**   | No                    | OTel                      | OTel                   | OTel                    | [OTel][]                 |
+| **Profiles** | No                    | No                        | No                     | No                      | Pyroscope                |
```

Comment:
Should we make Flow the first column in these tables? It seems to me more eye traffic will be on the left and also in case of narrow browser windows.

Reply:
@thampiotr I switched the order of the columns going with Flow, Static, Operator, OTel, Prometheus.